src/AWS.Deploy.Recipes/CdkTemplates/AspNetAppElasticBeanstalkLinux/Generated/UpdateBeanstalkEnvironment/OnEventLambda/OnEvent.py

```diff
@@ -0,0 +1,27 @@
+import boto3
+
+client = boto3.client('elasticbeanstalk')
+
+
+def handler(event, context):
+    request_type = event['RequestType'].lower()
+    if request_type == 'create' or request_type == 'update':
+        return on_create_or_update(event)
+    if request_type == 'delete':
```

Comment:
Add comment why we don't care about delete

Reply:
Fixed